### PR TITLE
[kube-prometheus-stack] Fix exemplar indentation for consistency

### PR DESCRIPTION
**Pull request by lpetrazickisupgrade**
This matches the indentation of everything else in the file. Existing template indents the final yaml by 2 spaces too many

Before:
```yaml
  exemplars:
      maxSize: 100000
```

After:
```yaml
  exemplars:
    maxSize: 100000
```

### CODE REVIEW

File: charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ spec:
 {{- end }}
 {{- if and (not .Values.prometheus.agentMode) .Values.prometheus.prometheusSpec.exemplars }}
   exemplars:
-  {{ toYaml .Values.prometheus.prometheusSpec.exemplars | indent 4 }}
+{{ toYaml .Values.prometheus.prometheusSpec.exemplars | indent 4 }}
```
Comment (GMartinez-Sisti) [inline charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml:106]: I think this is supposed to have 4 spaces to make it easier to read and understand it should be inside `exemplars`:
```suggestion
    {{ toYaml .Values.prometheus.prometheusSpec.exemplars | indent 4 }}
```

Comment (lpetrazickisupgrade) [reply charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml:106]: @GMartinez-Sisti I'm happy to add a dash and nindent, but I think it needs to be 2 spaces to match everything else in the template. Here is a larger snippet of rendered output from the current template:
```yaml
  externalLabels:
    cluster: dev
    environment: dev
  prometheusExternalLabelName: "prometheus"
  replicaExternalLabelName: "replica"
  enableRemoteWriteReceiver: true
  paused: false
  replicas: 2
  shards: 1
  logLevel:  info
  logFormat:  logfmt
  listenLocal: false
  enableAdminAPI: false
  exemplars:
      maxSize: 100000
  enableFeatures:
  - exemplar-storage
  scrapeInterval: 30s
  scrapeTimeout: 20s
  evaluationInterval: 1m
  resources:
    limits:
      memory: 6Gi
    requests:
      cpu: 300m
      memory: 6Gi
  retention: "10d"
  tsdb:
    outOfOrderTimeWindow: 0s
```

```suggestion
{{- toYaml .Values.prometheus.prometheusSpec.exemplars | nindent 2 }}
```

Comment (jkroepke) [reply charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml]: the indents are always absolute. the `{{-` will remove all whitespaces at the beginning, then the absolute indent is 4

Comment (lpetrazickisupgrade) [reply charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml]: @jkroepke Thank you, my bad! Updated to nindent by 4

**Review by jkroepke — DISMISSED** [review]

**Comment by jkroepke** [comment]
please bump the version on the Chart.yaml. thanks!

**Comment by lpetrazickisupgrade** [comment]
@jkroepke Thanks! Bumped

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**